lib/elixir/src/elixir_dispatch.erl

```diff
@@ -370,11 +370,11 @@ deprecation('Elixir.String', lstrip, 1) ->
 deprecation('Elixir.String', rstrip, 1) ->
   "use String.trim_trailing/1";
 deprecation('Elixir.String', strip, 2) ->
-  "use String.trim/2 with a binary second argument";
+  "use String.trim/2 with a binary as second argument";
 deprecation('Elixir.String', lstrip, 2) ->
-  "use String.trim_leading/2 with a binary second argument";
+  "use String.trim_leading/2 with a binary as second argument";
 deprecation('Elixir.String', rstrip, 2) ->
-  "use String.trim_trailing/2 with a binary second argument";
+  "use String.trim_trailing/2 with a binary as second argument";
```

Comment:
They mean slightly different things, "a binary second argument" means the "binary" is an adjective, while in " binary as second argument" "binary" is a noun. I'm fairly sure (as much as a fellow non-native speaker can be sure) that they're both fine. I don't have a preference though so if someone has, this change makes sense, otherwise eh :)